int8 output_id
bool state
---
bool response